src/Libraries/Nop.Data/Migrations/UpgradeTo470/AddIndexesMigration.cs

```diff
@@ -11,7 +11,7 @@
 
 namespace Nop.Data.Migrations.UpgradeTo470
 {
-    [NopSchemaMigration("2023-11-09 00:00:02", "AddIndexesMigration for 4.70.0")]
+    [NopSchemaMigration("2023-12-06 19:53:00", "Add Index For [Product].[Name]-[Forums_Topic].[Subject]")]
```

Comment:
I think that the description of migration should not be changed, let it remain the original one.